httsleep/main.py

```diff
@@ -32,7 +37,7 @@ def __init__(self, url_or_request, until=None, error=None,
         if ignore_exceptions:
             self.ignore_exceptions = tuple(ignore_exceptions)
         else:
-            self.ignore_exceptions = (None,)
+            self.ignore_exceptions = (_DummyException,)
```

Comment:
What about `self.ignore_exceptions = tuple()`? Don't know why I didn't do it like this originally...